pyclean/modern.py

```diff
@@ -90,6 +90,54 @@ def configure(self, args):
 Runner = CleanupRunner()
 
 
+def _normalize_pattern(pattern: str) -> str:
```

Comment:
Rename the function to make the code more natural to read:

```suggestion
def normalize_path(pattern: str) -> str:
```